tox.ini

```diff
@@ -1,3 +1,17 @@
 [flake8]
 max-line-length=120
 select = E,W,F,N,I
+
+[tox]
+envlist=py34,py35,flake8
+
+[tox:travis]
+3.4=py34
+3.5=py35,flake8
```

Comment:
I agree that in some cases flake8 suggests crazy constructions. But we can just add ignore to those lines and do a better formatting. 

I would leave it since it forces a generally better styling of code and normalised. So all contributors follow a generally equal style of coding.